ui/src/app/pages/apps-routes/app-installed-list/app-installed-list.page.ts

```diff
@@ -35,19 +35,15 @@ export class AppInstalledListPage extends Cleanup {
   segmentValue: 'services' | 'embassy' = 'services'
 
   showCertDownload : boolean
-  isConsulate: boolean
```

Comment:
is this just cleanup or is it a direct result of this change?

Reply:
clenaup